app/views/schools/users/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  self.page_title = "Manage Users"
```

Comment:
could this code go in a controller, rather than in the view?

Reply:
agreed to follow existing pattern 👍

Reply:
This decision was to maintain the current pattern in the codebase